tests/codegen.rs

```diff
@@ -249,3 +277,122 @@ pub struct TrustAnchor<'a> {
 }
 
 "#;
+
+impl CertificateMetadata {
```

Comment:
Can we keep the inherent impl right below the struct definition?

Reply:
Moved the impl.